src/main/java/InputData.java

```diff
@@ -0,0 +1,7 @@
+public interface InputData {
+
+        int debit (int balance, int debit);
+        int credit(int balance, int credit);
+        int debitMoreThenBalance(int balance, int credit, int debit);
```

Comment:
probably this method should be boolean and return true or false
in this case this method should look like
isDebitMoreThenBalance
or
isOperationValid for handling all possible cases